spec/fixtures/dfe_sign_in_service_approvers_response_page_1.json

```diff
@@ -5,27 +5,37 @@
         "id": "13F20E54-79EA-4146-8E39-18197576F023",
         "name": "Department for Education",
         "category": {
+          "id": "001",
           "name": "Establishment"
         },
         "type": {
+          "id": "03",
           "name": "Voluntary Aided School"
         },
         "urn": "000012",
+        "uid": "000012",
```

Comment:
It would make sense for the uid and urn to be different, so that if the method looks up uid when we want urn, the test would fail.